scripts/install.sh

```diff
@@ -35,59 +38,148 @@ function confirm () {
 }
 
 function composer_setup () {
+  echo "...Setting up Bene distro in $1"
   ROOT=$1
-  echo "...Setting up Bene distro in $ROOT"
-  composer create-project thinkshout/bene-project:master $ROOT --stability dev --no-interaction
+  # @TODO 04.11.2018 create-project pulls the WRONG VERSION of this project for some reason
+  # @TODO 04.11.2018 copying over pertinent files instead
+  # composer create-project thinkshout/bene-project:dev-4-fix-install $ROOT --stability dev --no-interaction
+  cp -R ./* $ROOT
+  cp ./.env.dist $ROOT
+}
+
+function project_setup () {
+  echo "...Configuring project"
+  composer install
+  composer drupal-scaffold
+  ./vendor/bin/robo init
+}
+
+function build_project () {
+  echo "...Building project"
+
+  echo -e "${INPUT_COLOR}Database name:${NO_COLOR}"
+  read db_name
+
+  DB_USER='root'
+  DB_PASS='root'
+  echo -e "${INPUT_COLOR}Use root:root for db?${NO_COLOR}"
+  DEFAULT_DB_SETTINGS=$(confirm '[Y/n]')
+  if [ "$DEFAULT_DB_SETTINGS" != "true" ]; then
+    echo -e "${INPUT_COLOR}Database user:${NO_COLOR}"
+    read db_user
+    DB_USER=$db_user
+
+    echo -e "${INPUT_COLOR}Database pass:${NO_COLOR}"
+    read db_pass
+    beneDB_PASS=$db_pass
+  fi
+
+  ./vendor/bin/robo configure --db-user=$DB_USER --db-pass=$DB_PASS --db-name=$db_name --profile=bene
+  ./vendor/bin/robo install
 }
 
 function git_setup () {
-  echo "Registering git repository"
+  echo "...Registering git repository"
   git init
-  echo "Git repo: "
+  echo -e "${INPUT_COLOR}Git repo [ex: git@github.com:thinkshout/bene.git]?${NO_COLOR}"
   read git_repo
   git remote add origin $git_repo
   git add .
-  echo "Inital commit message: " 
+  echo -e "${INPUT_COLOR}Initial commit message [ex: 'initial Bene installation']?${NO_COLOR}"
```

Comment:
I accidentally hit enter twice after the git repo question: that gave a blank input here and it errored out the whole script. Can we make it more fault-tolerant with either a default value or by asking the question again if the answer is empty?

Asking again if empty would be great for the git repo question too.